docroot/themes/humsci/humsci_basic/src/scss/components/_secondary-nav.scss

```diff
@@ -189,6 +189,32 @@
         }
       }
     }
+
+    // when a nolink item behaves as a toggle button, add button toggle icon
+    &.hb-secondary-toggler {
+      position: relative;
+
+      &::after {
+        @include hb-menu-icon-wrapper(48px, 48px);
+        @include hb-menu-plus-icon;
+        cursor: pointer;
```

Comment:
Should we consider moving `cursor: pointer` to line 196? This way, the user knows that they can click on the entire link item, but the functionality with the toggle remains the same.

Reply:
Good idea. I've made this change in the latest commit.